.travis.yml

```diff
@@ -1,17 +1,11 @@
 ---
 language: ruby
 bundler_args: "--without system"
-script: "bundle exec rspec --color --format documentation spec"
 notifications:
   email: false
 sudo: false
-jdk:
-  - openjdk11
```

Comment:
The docker tests don't need this? If not, looks good to me!

Reply:
Everything seems to run fine. I don't think docker depends on Java... I honestly don't know why we had that in here, might have been copy-pasta.